migrations/20250617204500-populate-fileuuids-in-thumbnails-batch.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+const MAX_ATTEMPTS = 10;
+const BATCH_SIZE = 1000;
+const SLEEP_TIME_MS = 5000;
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    let updatedCount = 0;
+    let attempts = 0;
+
+    const updateQuery = `
+      WITH batch AS (
+        SELECT t.id, f.uuid AS file_uuid
+        FROM thumbnails t
+        JOIN files f ON t.file_id = f.id
+        WHERE t.file_uuid IS NULL
+          AND t.file_id IS NOT NULL
+        LIMIT ${BATCH_SIZE}
+      )
+      UPDATE thumbnails
+      SET file_uuid = batch.file_uuid
+      FROM batch
+      WHERE thumbnails.id = batch.id
+      RETURNING thumbnails.id;
+    `;
+
+    console.info(
+      'Starting migration: populating thumbnails with missing file_uuid',
+    );
+
+    do {
+      try {
+        const [results] = await queryInterface.sequelize.query(updateQuery);
+        updatedCount = results.length;
+        attempts = 0;
+
+        console.info(`Updated ${updatedCount} thumbnails in this batch`);
```

Comment:
Use the accumulation (the total updated records) also please, thus I can follow the general progress